using-timescaledb/telemetry.md

```diff
@@ -0,0 +1,52 @@
+# Telemetry and Version Checking
```

Comment:
In this section, could we include the config file that you can manually change? This is to allow users who want to turn off the flag even before they CREATE EXTENSION.

Reply:
Reorganized this section after discussing w Sven